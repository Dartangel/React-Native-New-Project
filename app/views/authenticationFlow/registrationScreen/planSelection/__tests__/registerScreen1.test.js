import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import PlanSelection from '../planSelection';

describe('PlanSelection', () => {
    test('PlanSelection snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<PlanSelection />);
        expect(snapshot).toMatchSnapshot();
    });
});