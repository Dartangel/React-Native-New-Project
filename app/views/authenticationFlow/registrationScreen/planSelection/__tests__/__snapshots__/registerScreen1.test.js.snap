// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlanSelection PlanSelection snapshot 1`] = `
<ImageBackground
  source={
    Object {
      "testUri": "../../../app/src/img/splashScreen1.jpeg",
    }
  }
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Header
    header="Выберите Свой План"
    subHeader="Lose weight with an optimal workout plan"
  />
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <ChoosePlan
      subTitle="5-10 минут в день"
      title="Newbie"
    />
    <ChoosePlan
      subTitle="10-20 минут в день"
      title="Continuing"
    />
    <ChoosePlan
      subTitle="15-30 минут в день"
      title="Advanced"
    />
    <View
      style={
        Object {
          "alignItems": "center",
        }
      }
    >
      <NextButton
        onPress={[Function]}
        title="Next"
      />
    </View>
  </View>
</ImageBackground>
`;
